### PR TITLE
Use data from the main repo instead of CMC

### LINKED ISSUE #2: Stop using CMC
As suggested in https://github.com/atomiclabs/cryptocurrency-icons/issues/141#issuecomment-401698280

@imnotadeveloper Still interested in doing this?

### PR DESCRIPTION
**Pull request by pierreneter**
Close #2 
Related to https://github.com/atomiclabs/cryptocurrency-icons/pull/185

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -3,9 +3,8 @@ $(document).ready(function() {
   // ------------------------------
   // Settings
   // ------------------------------
-  var apiUrl       = 'https://api.coinmarketcap.com/v1/ticker/?limit=0';
-  var iconsBaseUrl = 'https://rawgit.com/atomiclabs/cryptocurrency-icons/master';
-  var dataJson     = 'https://raw.githubusercontent.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
+  var iconsBaseUrl = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master';
+  var dataJson     = 'https://cdn.rawgit.com/atomiclabs/cryptocurrency-icons/master/manifest.json';
```
Comment (sindresorhus) [inline js/main.js]: We can't use the CDN version of rawgit as it will not update when master updates.

Comment (pierreneter) [reply js/main.js]: ok

**Comment by sindresorhus** [comment]
I don't think you have tested this as it doesn't currently work. The symbol names have to be lowercased ;)

**Comment by sindresorhus** [comment]
I will fix after merge.

Thanks for the PR :)

**Pull request merged**